app/services/analytics.rb

```diff
@@ -237,9 +237,9 @@ def browser_attributes
   PROFILE_PERSONAL_KEY_VISIT = 'Profile: Visited new personal key'
   PROFILE_PERSONAL_KEY_CREATE = 'Profile: Created new personal key'
   PROFILE_PERSONAL_KEY_CREATE_NOTIFICATIONS = 'Profile: Created new personal key notifications'
-  PROOFING_ADDRESS_TIMEOUT = 'Proofing Address Timeout'
-  PROOFING_DOCUMENT_TIMEOUT = 'Proofing Document Timeout'
-  PROOFING_RESOLUTION_TIMEOUT = 'Proofing Resolution Timeout'
+  PROOFING_ADDRESS_RESULT_MISSING = 'Proofing Address Result Missing'
+  PROOFING_DOCUMENT_RESULT_MISSING = 'Proofing Document Result Missing'
+  PROOFING_RESOLUTION_RESULT_MISSING = 'Proofing Resolution Result Missing'
```

Comment:
to help us maintain continuity can we keep the old names as comments?

```suggestion
  PROOFING_ADDRESS_RESULT_MISSING = 'Proofing Address Result Missing' # Previously "Proofing Address Timeout"
  PROOFING_DOCUMENT_RESULT_MISSING = 'Proofing Document Result Missing' # Previously "Proofing Document Timeout"
  PROOFING_RESOLUTION_RESULT_MISSING = 'Proofing Resolution Result Missing' # Previously "Proofing Resolution Timeout"
```